src/strafenkatalog/strafenkatalog.html

```diff
@@ -13,7 +13,36 @@
         <li><a href="" class="active">Strafenkatalog</a></li>
     </ul>
 </nav>
-
 <!-- Content -->
+<!-- Tabelle_Strafenkatalog -->
+<div class="center">
+<table style="width:50%">
```

Comment:
Hier noch einmal die gesamte Tabelle einrücken. Also einfach vom open bis close tag markieren und auf tab drücken

Reply:
ach und du kannst mal schauen, ob anstatt des headers ein anderes Element nehmen könntest, welches du nicht extra mit einer größeren Schrift in CSS bearbeiten musst.
Hinweis: haben wir bereits im index.html benutzt